AsyncDisplayKit/ASDisplayNodeExtras.mm

```diff
@@ -251,13 +251,19 @@ static inline BOOL _ASDisplayNodeIsAncestorOfDisplayNode(ASDisplayNode *possible
 
 extern UIWindow * _Nullable ASFindWindowOfLayer(CALayer *layer)
 {
+  UIView *view = ASFindViewOfLayer(layer);
```

Comment:
`ASFindViewOfLayer` does that walking. I considered calling it `ASFindClosestViewOfLayer` of similar but decided to use the cleaner name and rely on documentation 😛